### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by drillsar**
Updated docs removed OpenGL; fixed broken link to the R600 table.

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ video drivers for your card.
 
 ### Windows
 
-* Windows 8 or 8.1
+* Windows 8/8.1 or Windows 10
```
Comment (gibbed) [inline docs/building.md]: It would be better to say "or later" here.

**Comment by IndieDriver** [comment]
While you're at it, you should fix the broken link to the R600 table in the gpu doc.
The table has been moved to [http://llvm.org/viewvc/llvm-project/llvm/trunk/lib/Target/AMDGPU/R600Instructions.td](http://llvm.org/viewvc/llvm-project/llvm/trunk/lib/Target/AMDGPU/R600Instructions.td)

**Comment by drillsar** [comment]
Ok I am fixing now

File: docs/building.md (hunk 1 above)
Comment (drillsar) [inline docs/building.md]: ok

File: docs/building.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ video drivers for your card.
 
 ### Windows
 
-* Windows 8 or 8.1
+* Windows 8 or later
 * Visual Studio 2015 or Visual Studio 2017
-* [Python 2.7](https://www.python.org/downloads/release/python-2713/)
+* [Python 3.6](https://www.python.org/downloads/release/python-362/)
```
Comment (DrChat) [inline docs/building.md]: AFAIK we still need python 2.7. Why did you up the version?

File: docs/gpu.md [hunk 3]
```diff
@@ -121,6 +121,6 @@ PM4 commands documented at [src/xenia/gpu/xenos.h](../src/xenia/gpu/xenos.h#L521
 
 ### Shaders
 
-* [LLVM R600 Tables](https://llvm.org/viewvc/llvm-project/llvm/trunk/lib/Target/R600/R600Instructions.td)
+* [LLVM R600 Tables]( http://llvm.org/viewvc/llvm-project/llvm/trunk/lib/Target/AMDGPU/R600Instructions.td)
```
Comment (DrChat) [inline docs/gpu.md]: Careful - there's a space before the link here.

File: docs/building.md (hunk 2 above)
Comment (drillsar) [inline docs/building.md]: @DrChat Python 3 is ready to be pulled I think. #755 pull request which I think needs 3.4+. I will change it than if need be.

File: docs/gpu.md (hunk 3 above)
Comment (drillsar) [inline docs/gpu.md]: Oops fixing that now

File: docs/building.md (hunk 2 above)
Comment (gibbed) [inline docs/building.md]: @drillsar PRs should be made with only the main repo in mind, not assuming other PRs have been pulled as well.

Comment (drillsar) [inline docs/building.md]: Ok I will change that now

**Comment by drillsar** [comment]
I didn't close this? I don't know what happened here

**Comment by drillsar** [comment]
@DrChat this good to go?

**Pull request merged**